code_gen/run.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+from code_gen.generate_component_code import main
+from code_gen.transform_code import transform
+from dotenv import load_dotenv
+load_dotenv()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--app', '-a', help='the app_name_slug', required=True)
+    parser.add_argument('--no-print', dest='noprint', help='do not print output to terminal', required=False, default=False, action='store_true')
+    parser.add_argument('prompt', help='the prompt to send to gpt-4, in between quotes')
+    args = parser.parse_args()
+
+    code = main(args.app, args.prompt, verbose=True)
```

Comment:
You can remove `verbose` here any anywhere you reference it in the code, use `logger.debug` to print logs in those cases. e.g.

```python
# create logger
logger = logging.getLogger(__name__)

# set log level based on debug flag
if args.debug:
    logger.setLevel(logging.DEBUG)
else:
    logger.setLevel(logging.WARNING)

# add console handler with a higher log level
handler = logging.StreamHandler()
handler.setLevel(logging.DEBUG)

# create formatter and add it to the handler
formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
handler.setFormatter(formatter)

# add the handlers to logger
logger.addHandler(handler)
```